.gitignore

```diff
@@ -1,13 +1,8 @@
-.idea
-node_modules
-package-lock.json
-animation.gif
-./package-lock.json
-dist
 .yarn/*
 !.yarn/patches
 !.yarn/releases
 !.yarn/plugins
 !.yarn/sdks
 !.yarn/versions
-.pnp.*
\ No newline at end of file
+dist
```

Comment:
If we want to not include dist anymore (makes sense as it seems like its only there as a high level "this works" check), then could we also remove the files which are already tracked from git repo? e.g.

```
git rm --cached dist
git add dist
git commit -m 'remove ignored files'
```